demos/chips.html

```diff
@@ -254,9 +254,8 @@ <h2>Custom theme</h2>
             entryInput.value = '';
           }
         };
-        ['click', 'keydown'].forEach(function(evtType) {
-          entryButton.addEventListener(evtType, addChip);
-        });
+        entryButton.addEventListener('click', addChip);
+        entryInput.addEventListener('keydown', addChip);
```

Comment:
Oh... What you really want is a `form` containing the text field and a submit button, which would make this Just Work without necessitating any keydown handling (though you'd want to `preventDefault` on form submit instead).

I won't insist on that being done in this PR, at least this does what was originally intended.

Reply:
Noted. I've been meaning to talk to a designer about making this input look better anyways, so I'll change it then.